CRM/Core/SelectValues.php

```diff
@@ -568,14 +568,7 @@ public static function contributionTokens(): array {
     foreach ($processor->getAllTokens() as $token => $title) {
       $tokens['{contribution.' . $token . '}'] = $title;
     }
-    return array_merge($tokens, [
-      '{contribution.cancel_reason}' => ts('Contribution Cancel Reason'),
-      '{contribution.amount_level}' => ts('Amount Level'),
-      '{contribution.check_number}' => ts('Check Number'),
-      '{contribution.campaign}' => ts('Contribution Campaign'),
-      // @todo - we shouldn't need to include custom fields here -
-      // remove, with test.
```

Comment:
The removed line is from the function that determins the tokens that show up here (per the comments I thought the line was redundant & UI testing should it to be true - I wound up UI testing rather than unit
testing that custom fields are still advertised 
as the test uses rollback & adding custom fields would break that. 

![image](https://user-images.githubusercontent.com/336308/128537119-dc94eab3-6caf-41d0-a624-b22f2ada5949.png)

![image](https://user-images.githubusercontent.com/336308/128537162-7ef33b1f-9233-43bd-8258-61ddb62df322.png)